src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15115,7 +15094,7 @@ namespace UnitarySystems {
             }
 
             state.dataCoilCooingDX->coilCoolingDXs[this->m_CoolingCoilIndex].simulate(
-                state, coilMode, CycRatio, this->m_CoolingSpeedNum, SpeedRatio, this->m_FanOpMode, singleMode, this->CoilSHR);
+                state, coilMode, this->m_CoolingSpeedNum, PartLoadFrac, this->m_FanOpMode, singleMode, this->CoilSHR);
```

Comment:
lines 15074 - 15087 are no longer needed (same for 15041 - 15068)

Reply:
Thanks; removed.